### PR TITLE
Add benchmarks for single-threaded Skwasm.

### PR DESCRIPTION
**Pull request by eyebrowsoffire**
This adds benchmarks that run on single threaded skwasm.

### CODE REVIEW

File: dev/benchmarks/macrobenchmarks/web/flutter_bootstrap.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
 {{flutter_build_config}}
+
+const searchParams = new URLSearchParams(window.location.search);
+const forceSt = searchParams.get('force_st');
+const extraConfig = forceSt ? {forceSingleThreadedSkwasm: true} : {};
 _flutter.loader.load({
   config: {
     // Use the local CanvasKit bundle instead of the CDN to reduce test flakiness.
     canvasKitBaseUrl: "/canvaskit/",
+    ...extraConfig,
```
Comment (ditman) [inline dev/benchmarks/macrobenchmarks/web/flutter_bootstrap.js:11]: Maybe:

```js
forceSingleThreadedSkwasm: !!forceSt
```

(Is there any difference between not passing the `forceSingleThreadedSkwasm` or making it `false`?)

**Review by ditman — APPROVED** [review]
LGTM! Where can we see the benchmark results?

**Comment by eyebrowsoffire** [comment]
@ditman 
> Where can we see the benchmark results?

We won't be able to see until postsubmit, since the builder doesn't exist yet. (also `presubmit: false`)

File: dev/benchmarks/macrobenchmarks/web/flutter_bootstrap.js (hunk 1 above)
Comment (eyebrowsoffire) [reply dev/benchmarks/macrobenchmarks/web/flutter_bootstrap.js:11]: I don't think it makes much of a difference either way, but I was hoping not to modify the config at all for existing benchmarks.

**Pull request merged**